samples/audioProfile.js

```diff
@@ -14,43 +14,38 @@
  */
 
 'use strict';
+function synthesizeSpeechAsync(request = {}) {
+  // Imports the Google Cloud client library
+  const textToSpeech = require('@google-cloud/text-to-speech');
+  // Creates a client
+  const client = new textToSpeech.TextToSpeechClient();
+  return new Promise((resolve, reject) => {
+    client.synthesizeSpeech(request, (err, response) => {
+      if (err) reject(err);
+      else resolve(response);
+    });
+  });
+}
 
-function synthesizeText(
+async function synthesizeText(
   text,
   outputFile,
   effectsProfileId,
   languageCode,
   ssmlGender
 ) {
   //[START tts_synthesize_text_audio_profile_beta]
-
-  // Imports the Google Cloud client library
-  const speech = require('@google-cloud/text-to-speech');
   const fs = require('fs');
 
-  // Creates a client
-  const client = new speech.TextToSpeechClient();
-
   const request = {
     input: {text: text},
     voice: {languageCode: languageCode, ssmlGender: ssmlGender},
     audioConfig: {audioEncoding: 'MP3', effectsProfileId: effectsProfileId},
   };
 
-  client.synthesizeSpeech(request, (err, response) => {
-    if (err) {
-      console.error(`ERROR:`, err);
-      return;
-    }
-
-    fs.writeFile(outputFile, response.audioContent, 'binary', err => {
-      if (err) {
-        console.error('ERROR:', err);
-        return;
-      }
-      console.log(`Audio content written to file: ${outputFile}`);
-    });
-  });
+  const response = await synthesizeSpeechAsync(request);
+  fs.writeFileSync(outputFile, response.audioContent, 'binary');
```

Comment:
Instead of using `Sync` methods, please do something like this:
```js
const writeFile = util.promisify(fs.writeFile);
await writeFile(/* params */);
```

Reply:
Great! and nice way to do, implemented requested changes.